### PR TITLE
Improve tooltip positions

### PR DESCRIPTION
**Pull request by toolmantim**
This updates the tooltip position and graph styling.

- [ ] Fix grey lines missing the gap
- [ ] Fix hover on gap in the bars causing a re-render of tooltip

Before:

tooltips 1

After:

tooltips 2

### CODE REVIEW

**Comment by toolmantim** [comment]
The tooltip on the build status could do with an animation.

**Comment by keithpitt** [comment]
Nice! Done so soon!

I'll review first thing in the morn 👍

**Comment by keithpitt** [comment]
So much better not moving with the mouse

File: app/components/organization/Pipeline/bar.js [hunk 1]
```diff
@@ -1,42 +1,74 @@
...
+  }
+
+  handleMouseOut = () => {
+    this.setState({hover: false})
+  }
 }
 
-export default Bar
+
+export default Relay.createContainer(Bar, {
```
Comment (keithpitt) [inline app/components/organization/Pipeline/bar.js]: We might need to roll back this change if having this many relay containers on the page makes it go cray cray.

File: app/components/organization/Pipeline/build-tooltip.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  }
+
+  renderTime() {
+    if(this.props.build.startedAt || this.props.build.finishedAt) {
+      return (
+        <FriendlyTime value={this.props.build.finishedAt || this.props.build.startedAt} />
+      )
+    } else {
+      return (
+        <div>something</div>
```
Comment (keithpitt) [inline app/components/organization/Pipeline/build-tooltip.js]: something hey?

Comment (toolmantim) [inline app/components/organization/Pipeline/build-tooltip.js]: That was yours! I've just left it in :D :D This PR doesn't change the content of the tooltips, just the positioning

File: app/components/organization/Pipeline/bar.js (hunk 1 above)
Comment (toolmantim) [inline app/components/organization/Pipeline/bar.js]: Oh this was meant to be taken out!

File: app/components/organization/Pipeline/build-tooltip.js (hunk 2 above)
Comment (keithpitt) [inline app/components/organization/Pipeline/build-tooltip.js]: Oh right, ha ha! We'll fix it when we tweak the contents

**Comment by toolmantim** [comment]
This PR still needs a few things

**Pull request merged**